web/src/components/input-label/index.tsx

```diff
@@ -32,21 +34,19 @@ export const InputLabel: React.FC<Props> = props => {
     theme,
   })
   const hasLabel = label && label !== ''
-  const Wrapper = hasLabel ? Text : 'div'
+
   return (
-    <Wrapper
-      {...(hasLabel && {
-        size: 'base',
-        as: 'label',
-        color: isDisabled ? 'secondary' : 'primary',
-        htmlFor: id,
-      })}
+    <Text
+      size="base"
+      as="label"
+      color={isDisabled ? 'secondary' : 'primary'}
+      htmlFor={id}
     >
       <VisuallyHidden>{renderHiddenInput?.()}</VisuallyHidden>
 
       <div className={classes.label}>
         {renderInput?.()}
-        {hasLabel && label}
+        {hasLabel && <span id={labelId}>{label}</span>}
```

Comment:
@pseudonymcp @AleshaOleg  the spacing between the labels is only because of the `line-height` of the label (no margin or padding). When using the switch without labels (or empty lables) they lose the spacing and stick to each other:
![Screenshot 2021-02-02 at 14 08 34](https://user-images.githubusercontent.com/385177/106605414-2a3ff600-6561-11eb-9293-2e60f1d52cd5.png)


I'm not so sure if this is a problem. my CSS knowledge is limited

Reply:
It's the same as for the checkbox, isn't it? The surrounding margin of components shouldn't be part of the component itself, so this looks ok I guess. In the interface we should use spacers to position the components.

Reply:
makes sense